parachains/integration-tests/e2e/assets/asset-hub-kusama/0_xcm/2_ump.yml

```diff
@@ -97,10 +97,17 @@ tests:
                     - name: polkadotXcm.Attempted
                       threshold: *weight_threshold
                       result: [{ Complete: { refTime: '539,494,000', proofSize: '7,133' }}]
-                    - name: ump.ExecutedUpward
+                    - name: messageQueue.Processed
                       chain: *relay_chain
                       threshold: *weight_threshold
-                      result: [{ Complete: { refTime: '298,716,000', proofSize: 0 }}]
+                      result:
+                        origin:
+                        Ump:
+                          Para: '1,000'
+                        weightUsed:
+                          refTime: '298,716,000'
+                          proofSize: '0'
+                        success: true
```

Comment:
Do you mind to use the string format for `results` as it is done for the rest of tests to be consistent?

Reply:
will be in next PR.